docs/users-guide/custom-questions.md

```diff
@@ -0,0 +1,126 @@
+## Creating custom questions with the notebook editor
+
+---
+
+If you have a question that's a bit more involved than a [simple question](04-asking-questions.md), you can create a custom question using the notebook editor. You can get there by clicking the Ask a Question button in the top nav bar and selecting Custom Question. If you're on a question detail page, you can also open up the editor by clicking the icon in the top right of the screen.
+
+### The parts of the notebook
+
+![The notebook](./images/notebook/notebook-ui.png)
+
+The notebook is made up of a sequence of individual steps. Under each step you'll see buttons to add more steps after the current one. To the right of each step is a preview button that shows you the first 10 rows of the results of your question up to that step.
+
+![Previewing results](./images/notebook/preview-table.png)
+
+#### Picking your starting data
+
+This first step is required, and is where you pick the data that you want to base your question on. In most cases you'll pick one of the tables in your database, but you can also choose a previously saved question's result as the starting point for your new question. What this means in practice is that you can do things like use complex SQL queries to create new tables that can be used as starting data in a question just like any other table in your database.
+
+You can use most saved questions as source data, provided you have [permission](../administration-guide/05-setting-permissions.html) to view that question. You can even use questions that were saved as a chart rather than a table.
+
+There are some kinds of saved questions that can't be used as source data:
+
+- Druid questions
+- Google Analytics questions
+- Mongo questions
+- questions that use `Cumulative Sum` or `Cumulative Count` aggregations
+- questions that have columns that are named the same or similar thing, like `Count` and `Count 2`
```

Comment:
I don't under the last bullet. I don't know about this limitation.

Reply:
Yeah, like if your question has a count of rows and a cumulative count of rows, the columns are named `count` and `count`, and it won't let you use it as starting data. I just tried this out and it's still a limitation.